domain/src/main/java/org/oppia/android/domain/spotlight/SpotlightStateController.kt

```diff
@@ -58,10 +66,8 @@ class SpotlightStateController @Inject constructor(
 
   /**
    * Retrieves the current [SpotlightViewState] of a spotlit feature for a given profile.
-   *
```

Comment:
This one should be kept--there's always a blank doc line between the summary and the at-clauses.

Reply:
done